Makefile

```diff
@@ -12,6 +12,18 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
+UNAME_ARCH     := $(shell uname -m)
+ifeq ($(UNAME_ARCH),x86_64)
+	TEMP_ARCH = amd64
+else ifeq ($(UNAME_ARCH),aarch64)
+	TEMP_ARCH = arm64
+else
+	TEMP_ARCH := $(UNAME_ARCH)
+endif
+
+GOARCH ?= $(TEMP_ARCH)
+GOOS = linux
+
```

Comment:
I would drop this whole part, since go already defaults to the host arch when `GOARCH` is unset.

Reply:
Not sure you're looking at the latest version of this PR (:

Reply:
can we move the`GIT_HASH` to `hack/build/build-push-multiarch-images.sh`

Reply:
I can. I need to fix it anyway, because I didn't actually passed it to the script. I'll fix.

The question is if we want it for the regular build script as well?